### DISCUSSION TITLE
Welcome to cntext Discussions!

### DISCUSSION THREAD

**Discussion by hiDaDeng** [hiDaDeng/cntext#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hiibii** [comment]
Hi, thanks for sharing this helpful tool! I'm using it to analyze Chinese texts in my research project.

I have a question regarding the readability score. You referred to Xu et al. (2019) for computing the three readability scores for Chinese texts, where readability1 is the average word number in each clause. Here, clauses should refer to "subsentences", instead of sentences, right?
 
For example, the readability1 score for "如何看待一网文作者被黑客大佬盗号改文，因万分惭愧而停更。"  should be (18+8)/2, which is 13. 
However, the current model outputs 28 as the readability1 score. That is, the model now computes the average number of words in an entire sentence as readability1.  Could you please help resolve the issue? Thanks a lot!

<img width="1099" alt="image" src="https://github.com/hiDaDeng/cntext/assets/49041686/960762f2-01bb-4019-9005-0b4b9dcae5f3">

**Comment by lylyl12123** [comment]
你好，我在用的时候发现了一些问题，可读性2的定义文档中是说每个子句中连词所占的比例。但我在测试"今天天气很好，我们一起去公园玩。",这个句子的时候，发现r2: 2.0，明显不是在0-1之间的比例。然后我翻阅了源代码，发现代码中的计算逻辑是，计算平均每句连词副词个数。是否是您在更新代码的时候没有更新文档呢？
